package.json

```diff
@@ -30,7 +30,7 @@
     "autoprefixer": "8.6.5",
     "autosize": "4.0.2",
     "babel-core": "7.0.0-bridge.0",
-    "babel-jest": "22.4.1",
```

Comment:
It's a bit surprising to me that this is in `dependencies` rather than `devDependencies`. Do you know whether that's intentional?

Reply:
Our approach is that packages that are needed to build the production image are in `dependencies`, even if they are build tools that are not part of the compiled output.

Only the packages that are not needed for production build, e.g., linting and testing tools, should be `devDependencies`.

That means you're right: if I'm not overlooking something, `babel-jest` should indeed be in `devDependencies`.

The Babel Core Bridge also looks interesting:
```
"babel-core": "7.0.0-bridge.0"
```
Is that for Jest integration only?

Reply:
@jsnajdr I only see one place where `babel-core` is pulled in, the babel cache identifier creator. I'm not sure it's related to jest.

Reply:
I'm trying to update the cache key creator and remove the `babel-core` dep. It seems to work. PR coming shortly.